test/GitIgnoreTest.php

```diff
@@ -91,13 +119,22 @@ public function testAddEntryDoesNotAddFileOrDirectoryIfItIsInsideAnIgnoredDirect
 
     public function testIgnoreDirectoriesAreNotWrittenIfNoEntriesAreAdded()
```

Comment:
My reasoning is, say, you decide to change the ignored directories in your `composer.json` file... The only way to have that actually take effect is to empty those lines from your `.gitignore` file, remove Magento's files and reinstall.

This hasn't actually hit me though so I'm not really worried if we don't include it. I just saw it as an opportunity to solve an issue before somebody gets stumped by that edge-case.